File: Write_test/guanglianda/3.cpp
// n = int(input())
// nums = list(map(int, input().split(' ')))
// max_num = nums[0]
// result = 0
// for num in nums:
//     if num < max_num:
//         result += 1
//     else:
//         max_num = num
// print(result)
#include <iostream>
#include <array>
#include <string>
#include <vector>
using namespace std;
int main()
{
    int n;
    cin>>n;
    vector<int> nums;
    for(int i=0;i<n;i++){
        int temp;
        cin>>temp;
        nums.push_back(temp);
    }
    int maxnum=nums[0];
    int reuslt=0;
    for(auto num:nums){
        if (num<maxnum){
            reuslt+=1;
        }else{
            maxnum=num;
        }
    }
    cout<<reuslt;
    return 0;
}

File: Write_test/wangyi/2021C++/3.py
# 7 3 1 4
#10 20 2 29 ——20+29
nums = list(map(int, input().split()))

nums2 = {1: 0, 2: 0, 3: 0, 4: 0, 5: 0, 6: 0}
res = 0
for num in nums:
    if num % 7 == 0:
        res += num
    else:
        res += (num//7)*7
        nums2[num % 7] += 1

for i in range(1, 4):
    minnum = min(nums2[i], nums2[7-i])
    nums2[i] -= minnum
    nums2[7-i] -= minnum
    res += 7*minnum
if res==0:
    print(-1)
else:
    print(res)
